traitsui/testing/tester/default_registry.py

```diff
@@ -0,0 +1,34 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+
+import importlib
+
+from traitsui.testing.tester.registry import TargetRegistry
+
+def get_default_registry():
+    """ Creates a default registry for UITester that is toolkit specific.
+
+    Returns
+    -------
+    registry : TargetRegistry
+        The default registry containing implementations for TraitsUI editors
+        that is toolkit specific.  
+    """
+    # side-effect to determine current toolkit
+    from pyface.toolkit import toolkit_object
+    from traits.etsconfig.api import ETSConfig
+
+    if ETSConfig.toolkit == "null":
+        return TargetRegistry()
+    else:
+        toolkit = {'wx': 'wx', 'qt4': 'qt4', 'qt': 'qt4'}[ETSConfig.toolkit]
```

Comment:
There may be a better way to map qt to qt4

Reply:
Looks fine to me. This is logically the same as defining an entry point to map "qt" to the "qt4" package like how it is done in traitsui and pyface setup.py.